test/evergreen.yml

```diff
@@ -136,6 +136,33 @@ functions:
           DEFINED_EVERGREEN_CONFIG_FLAGS=${DEFINED_EVERGREEN_CONFIG_FLAGS/\-DHAVE_BUILTIN_EXTENSION_ZSTD=1/}
         fi
 
+        if [ "${build_variant|}" = "macos-1100" ]; then
+          # For mac builds, we want explicitly tell cmake which python to use, as
+          # well as the matching library directory and header files. The find_libpython
+          # module gives us the library.
+          SYSPY=${python_binary}
+          $SYSPY -mvenv venv
+          source venv/bin/activate
+          pip install find_libpython
```

Comment:
A future-proofing comment: Later version of macOS may require you to specify pip3 instead of pip, but if it works for 11, then feel free to keep it this way!

Reply:
Fixed